### PR TITLE
Update README.md 

### PR DESCRIPTION
**Pull request by dybi**
resolves: https://github.com/golemfactory/concent/issues/947

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ You can put the output of the script above directly in your `local_settings.py`
 
 ### Preparing your environment for development
 
-1. Install [Python](https://www.python.org/) >= 3.5 using your package manager.
+1. Install [Python](https://www.python.org/) >= 3.6 using your package manager.
```
Comment (cameel) [inline README.md]: This is unrelated. Should be a separate commit.

Comment (dybi) [reply README.md]: ok
